src/app/home/home.component.html

```diff
@@ -1,93 +1,67 @@
-<div id="home" class="container content">
-    <div class="row">
-        <div class="col-md-4">
-        </div>
-        <div class="col-md-2">
-            <a class="btn btn-default btn-block" type="button" href="https://fabric8.io/guide/getStarted/index.html">Learn</a>
-        </div>
-        <div class="col-md-2">
-            <button class="btn btn-default btn-block" type="button" (click)="createSpace.open()">
-                Start Collaborating
-            </button>
-        </div>
-        <div class="col-md-4"></div>
+<div class="container-fluid container-cards-pf">
```

Comment:
yes .. just took a stab at it to make for more space for content in desktop usage. If we set a max width we can go back to being centered.